test/runtests.jl

```diff
@@ -1,6 +1,23 @@
 using RecurrenceAnalysis
 using Test
 
+# Download some test timeseries
+# Old:
+repo = "https://raw.githubusercontent.com/JuliaDynamics/NonlinearDynamicsTextbook/master/exercise_data"
+tsfolder = joinpath(@__DIR__, "timeseries")
+todownload = ["$n.csv" for n in 1:4]
```

Comment:
Maybe we should put these test in the JuliaDynamics/JuliaDynamics repo? The same repo that hosts videos and/or images for documentation.

Reply:
You want to put the tests into that repo? Why?

Reply:
Because this repo is associated with our book project. Its contents might change in the second edition. Speculative stuff.

Reply:
I see. Pretty speculative stuff, indeed. Okay, now I get your point: You want to migrate the timeseries stored in "https://raw.githubusercontent.com/JuliaDynamics/NonlinearDynamicsTextbook/master/exercise_data" into "https://raw.githubusercontent.com/JuliaDynamics/JuliaDynamics/master/timeseries"? That makes sense. I was only confused because I thought you mean moving the actual tests into that repo. Am I getting this right now? Then we also have to change tests in `DelayEmbeddings.jl`, I can make a new PR for all that separately.

Reply:
Hahah yes sorry, I only meant to transfer the .csv files to the second link that you posted!

Reply:
@hkraemer you have already moved the timeseries to the other repo, right? Then we have to change the link above,